### PR TITLE
Updated examples for models to be more logical

### PR DESCRIPTION
**Pull request by gewenyu99**
## What does this PR do?

The examples used in documentation for many of our models are not clear. This tries to make the examples more logical and context appropriate.

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

**Comment by gewenyu99** [comment]
Another question: is this wrong? The read-write permissions are strings?
```php
  ->addRule('$read', [
                'type' => self::TYPE_STRING,
                'description' => 'Execution read permissions.',
                'default' => '',
                'example' => 'role:all',
                'array' => true,
            ])
 ```

File: src/Appwrite/Utopia/Response/Model/AttributeEnum.php [hunk 1]
```diff
@@ -28,7 +28,7 @@ public function __construct()
                 'type' => self::TYPE_STRING,
                 'description' => 'Array of elements in enumerated type.',
                 'default' => null,
-                'example' => 'element',
+                'example' => "['elementA','elementB']",
```
Comment (eldadfux) [inline src/Appwrite/Utopia/Response/Model/AttributeEnum.php:31]: Are you sure this is the right way to give an example of an array? I was sure that on the docs we just build the array from the single value example, but I see a lot of inconsistency, let's double check this and make sure we are consistent about the type here.

File: src/Appwrite/Utopia/Response/Model/Platform.php [hunk 2]
```diff
@@ -60,7 +60,7 @@ public function __construct()
                 'type' => self::TYPE_STRING,
                 'description' => 'Web app hostname. Empty string for other platforms.',
                 'default' => '',
-                'example' => true,
+                'example' => 'demo.appwrite.com',
```
Comment (eldadfux) [inline src/Appwrite/Utopia/Response/Model/Platform.php:63]:
```suggestion
                'example' => 'myapp.appwrite.io',
```

**Review by eldadfux — CHANGES_REQUESTED** [review]

File: src/Appwrite/Utopia/Response/Model/AttributeEnum.php (hunk 1 above)
Comment (gewenyu99) [reply src/Appwrite/Utopia/Response/Model/AttributeEnum.php:31]: Yeah I wasn't sure either 😑 Let me build a preview and check.

File: src/Appwrite/Utopia/Response/Model/Execution.php [hunk 3]
```diff
@@ -63,13 +63,13 @@ public function __construct()
                 'type' => self::TYPE_STRING,
                 'description' => 'The script response output string. Logs the last 4,000 characters of the execution response output.',
                 'default' => '',
-                'example' => '',
+                'example' => '{"areDevelopersAwesome":"true"}',
```
[the remark this answers is not in the document]
Comment (gewenyu99) [reply src/Appwrite/Utopia/Response/Model/Execution.php:66]: Screen Shot 2022-08-10 at 4 50 56 PM

I think it comes back as a JSON string when I build the preview. I think this is a good way to demonstrate that 1. it's a string, 2. you can return JSON strings.

For example, this is what's returned by one of my functions:
```json
{"total":1,"executions":[{"$id":"62f3d83546d3653636de","$createdAt":1660147765,"$updatedAt":1660147768,"$read":["user:61e71a5ba884a3277580"],"functionId":"62c3843c0c8cf826add7","trigger":"http","status":"completed","statusCode":200,
"response":"{\"env\":{\"APPWRITE_FUNCTION_ID\":\"62c3843c0c8cf826add7\",\"APPWRITE_FUNCTION_NAME\":\"test\",\"APPWRITE_FUNCTION_DEPLOYMENT\":\"62c3845c23423d015472\",\"APPWRITE_FUNCTION_RUNTIME_NAME\":\"Node.js\",\"APPWRITE_FUNCTION_RUNTIME_VERSION\":\"16.0\",\"APPWRITE_FUNCTION_TRIGGER\":\"http\",\"APPWRITE_FUNCTION_EVENT\":null,\"APPWRITE_FUNCTION_EVENT_DATA\":null,\"APPWRITE_FUNCTION_DATA\":\"dsaf\",\"APPWRITE_FUNCTION_PROJECT_ID\":\"how-do-i-index\",\"APPWRITE_FUNCTION_USER_ID\":\"61e71a5ba884a3277580\",\"APPWRITE_FUNCTION_JWT\":\"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VySWQiOiI2MWU3MWE1YmE4ODRhMzI3NzU4MCIsInNlc3Npb25JZCI6IjYyODdhNTEzMjBkY2FhNDU0YTBiIiwiZXhwIjoxNjYwMTQ4NjY1fQ.TUMnsWcTLM2S9r4FHjjXlm6ulWjzhpjhU8SxpkbzCAs\"},\"headers\":{},\"payload\":\"dsaf\"}","stderr":"","time":0.043705}]}
```

File: src/Appwrite/Utopia/Response/Model/AttributeEnum.php (hunk 1 above)
Comment (gewenyu99) [reply src/Appwrite/Utopia/Response/Model/AttributeEnum.php:31]: Yep this is wrong. I need to fix these and check them manually.

File: src/Appwrite/Utopia/Response/Model/Index.php [hunk 4]
```diff
@@ -32,14 +32,14 @@ public function __construct()
                 'type' => self::TYPE_STRING,
                 'description' => 'Index attributes.',
                 'default' => [],
-                'example' => [],
+                'example' => ["title", "description"],
```
Comment (gewenyu99) [inline src/Appwrite/Utopia/Response/Model/Index.php:35]: Screen Shot 2022-08-11 at 1 57 59 PM

File: src/Appwrite/Utopia/Response/Model/File.php [hunk 5]
```diff
@@ -38,14 +38,14 @@ public function __construct()
                 'type' => self::TYPE_STRING,
                 'description' => 'File read permissions.',
                 'default' => [],
-                'example' => 'role:all',
+                'example' => ['role:all'],
```
Comment (gewenyu99) [inline src/Appwrite/Utopia/Response/Model/File.php:41]: Screen Shot 2022-08-11 at 1 58 29 PM

File: src/Appwrite/Utopia/Response/Model/AttributeEnum.php (hunk 1 above)
Comment (gewenyu99) [reply src/Appwrite/Utopia/Response/Model/AttributeEnum.php:31]: Screen Shot 2022-08-11 at 2 19 10 PM
I found the proper syntax :)

File: src/Appwrite/Utopia/Response/Model/File.php (hunk 5 above)
Comment (gewenyu99) [reply src/Appwrite/Utopia/Response/Model/File.php:41]: Although that mime type looks super weird.

File: src/Appwrite/Utopia/Response/Model/Membership.php [hunk 6]
```diff
@@ -80,7 +80,7 @@ public function __construct()
                 'type' => self::TYPE_STRING,
                 'description' => 'User list of roles',
                 'default' => [],
-                'example' => 'admin',
+                'example' => ['admin'],
```
Comment (gewenyu99) [inline src/Appwrite/Utopia/Response/Model/Membership.php:83]: Screen Shot 2022-08-11 at 2 23 17 PM

File: src/Appwrite/Utopia/Response/Model/Webhook.php [hunk 7]
```diff
@@ -49,7 +49,7 @@ public function __construct()
                 'type' => self::TYPE_STRING,
                 'description' => 'Webhook trigger events.',
                 'default' => [],
-                'example' => 'database.collections.update',
+                'example' => ['databases.*.collections.*.update'],
```
Comment (gewenyu99) [inline src/Appwrite/Utopia/Response/Model/Webhook.php:52]: Screen Shot 2022-08-11 at 2 24 17 PM
Again the example works, but the links are escaped weirdly

**Comment by lohanidamodar** [comment]
@stnguyen90 if this is still relevant please sync and update, if not, please close.

**Comment by gewenyu99** [comment]
@lohanidamodar Thanks for reminding me. These need to be redone, but with generated examples elsewhere. This implementation is no longer relevant. Closing

**Pull request closed without merging**